package.json

```diff
@@ -4,12 +4,12 @@
   "description": "User interface for the Knowledge Based Verification (KBV) part of the Identity Proofing and Verification (IPV) system within the GDS digital identity platform, GOV.UK Sign In.",
   "main": "src/server.js",
   "engines": {
-    "node": "16.*",
+    "node": "18.*",
     "yarn": "1.22.*"
   },
   "scripts": {
     "start": "node src/app.js",
-    "start:ci": "API_BASE_URL=http://localhost:8090 REDIS_SESSION_URL= NODE_ENV=development node src/app.js",
+    "start:ci": "API_BASE_URL=http://127.0.0.1:8090 REDIS_SESSION_URL= NODE_ENV=development node src/app.js",
```

Comment:
Our wiremock api starts in the IPV4 world. The simplest solution is to tell it to use `127..`